### ISSUE TITLE
No module named `simvg.models.fusions`

### ISSUE THREAD

**Issue by moqj** [Dmmm1997/SimVG#2]
Hello,

First of all, thank you for the excellent work on SimVG! I am interested in this project and am currently trying to reproduce the results. 

However, I encountered an issue during implementation. When running the code, I got the following error:
```
Traceback (most recent call last):
  File "tools/train.py", line 13, in <module>
    from simvg.models import build_model, ExponentialMovingAverage
  File "/home/gpu18/workspace/SimVG/simvg/models/__init__.py", line 4, in <module>
    from .fusions import *
ModuleNotFoundError: No module named 'simvg.models.fusions
```

I have checked the project structure but couldn't locate the `fusions` module. Could you kindly provide guidance on whether this is a missing file or if I might have missed some configuration steps?

**Comment by Dmmm1997** [comment]
> Hello,
> 
> First of all, thank you for the excellent work on SimVG! I am interested in this project and am currently trying to reproduce the results.
> 
> However, I encountered an issue during implementation. When running the code, I got the following error:
> 
> ```
> Traceback (most recent call last):
>   File "tools/train.py", line 13, in <module>
>     from simvg.models import build_model, ExponentialMovingAverage
>   File "/home/gpu18/workspace/SimVG/simvg/models/__init__.py", line 4, in <module>
>     from .fusions import *
> ModuleNotFoundError: No module named 'simvg.models.fusions
> ```
> 
> I have checked the project structure but couldn't locate the `fusions` module. Could you kindly provide guidance on whether this is a missing file or if I might have missed some configuration steps?

Thank for your attention.
We have fixed this issue in the newest comment b7319ca5f24cd3f46cbe566010288f83b8c9268b.

**Comment by moqj** [comment]
Thanks!
